build.gradle

```diff
@@ -208,7 +209,11 @@ dependencies {
     provided "appeng:RotaryCraft:${config.rotc.version}:api"
     provided ("appeng:appliedenergistics2:${config.ae2.version}:dev") {
         exclude module: 'buildcraft'
-    }
+    }*/
+
```

Comment:
I've added a comment on the main conversation thread with alternatives, if there are other non-api usages you can't work out I'd be happy to assist